exegol/utils/SoundUtils.py

```diff
@@ -0,0 +1,48 @@
+import io
+import os
+import shutil
+import subprocess
+import time
+from pathlib import Path
+from typing import Optional
+
+from exegol.console.ExegolPrompt import Confirm
+from exegol.exceptions.ExegolExceptions import CancelOperation
+from exegol.utils.EnvInfo import EnvInfo
+from exegol.utils.ExeLog import logger, console
+
+
+class SoundUtils:
+    """This utility class allows determining if the current system supports the sound sharing
+    from the information of the system."""
+
+    __distro_name = ""
+
+    @classmethod
+    def isPulseAudioAvailable(cls) -> bool:
+        """
+        Check if the host OS has PulseAudio installed
+        :return: bool
+        """
+        assert os.getenv("XDG_RUNTIME_DIR")
```

Comment:
Instead of using an `assert` and crashing the wrapper, prefer using a fallback like `os.getenv("XDG_RUNTIME_DIR",f"<Default_Path>")`

Or `return False` is this env is not available and you cant supply a fallback.

Reply:
Changed to a fallback with the default path.